rs/bitcoin/ckbtc/minter/src/updates/tests.rs

```diff
@@ -217,6 +204,93 @@ mod update_balance {
             ],
         );
     }
+
+    #[tokio::test]
+    async fn should_observe_latency_metrics() {
+        init_state_with_ecdsa_public_key();
+
+        async fn update_balance_with_latency(
+            latency: Duration,
+            account_utxos: Vec<Utxo>,
+        ) -> Result<Vec<UtxoStatus>, UpdateBalanceError> {
+            let account = ledger_account();
+            let update_balance_args = UpdateBalanceArgs {
+                owner: Some(account.owner),
+                subaccount: account.subaccount,
+            };
+            let mut runtime = MockCanisterRuntime::new();
+            mock_schedule_now_process_logic(&mut runtime);
+            mock_get_utxos_for_account(&mut runtime, account, account_utxos);
+            mock_time(
+                &mut runtime,
+                vec![
+                    NOW,                         // start time of `update_balance` method
+                    NOW,                         // time used to process UTXOs
```

Comment:
nit: slight reformulation since otherwise I have the impression that processing UTXOs is instantaneous.
```suggestion
                    NOW,                         // time used to triage processable UTXOs
```